src/index.html

```diff
@@ -21,7 +21,347 @@
+        </div>
+        <button class="card__button">Buy</button>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Macbook"
+        />
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="card__stars stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__reviews-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Macbook"
+        />
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="card__stars stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__reviews-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </section>
+      <section class="card">
+        <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="Macbook"
+        />
+
+        <h2 class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__reviews">
+          <div class="card__stars stars">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+
+          <p class="card__reviews-reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+        <button class="card__button">Buy</button>
+      </section>
```

Comment:
The product card sections are repeated multiple times with the same content. Consider using a templating engine or JavaScript to dynamically generate these sections to avoid redundancy and make maintenance easier.